camera1:
  topic_image: /camera1/color/image_raw
  topic_image_camera_info: /camera1/color/camera_info
  topic_depth: /camera1/aligned_depth_to_color/image_raw
  topic_depth_camera_info: /camera1/aligned_depth_to_color/camera_info
  depth_scale: 0.000125
  camera_extrinsic: [-0.67071412,  0.66646329, -0.32552919,  0.602624374719,
                      0.74161591,  0.59537533, -0.30908584,  0.454415090715,
                     -0.01218232, -0.44872586, -0.89358642,  0.327708692065,
                      0.        ,  0.        ,  0.        ,  1.        ]
  workspace: [0.43, 0.34, -0.095, 0.6123724, 0.6123724, -0.3535534, -0.3535534, 0.44, 0.22, 0.30]
  param_segmentation: "remove_background=0,workspace=0.43 0.34 -0.095 0.6123724 0.6123724 -0.3535534 -0.3535534 0.44 0.22 0.30"
camera2:
  topic_image: /camera2/color/image_raw
  topic_image_camera_info: /camera2/color/camera_info
  topic_depth: /camera2/aligned_depth_to_color/image_raw
  topic_depth_camera_info: /camera2/aligned_depth_to_color/camera_info
  depth_scale: 0.000125
  camera_extrinsic: [0.596874342704, -0.44576608171, 0.324836952587, -0.199589661547, 0.875581321924, 0.430836309741, -0.11645117846]
  #workspace: [0.418, -0.355, -0.140, 0.609, 0.609, 0.360, 0.360, 0.32, 0.12, 0.244]
  workspace: [0.418, -0.355, -0.140, 0.866, 0.000, 0.000, 0.500, 0.32, 0.244, 0.12]
  param_segmentation: "remove_background=0,workspace=0.418 -0.355 -0.140 0.609 0.609 0.360 0.360 0.32 0.12 0.244"